core/web/resolver/p2p.go

```diff
@@ -0,0 +1,96 @@
+package resolver
+
+import "github.com/smartcontractkit/chainlink/core/services/keystore/keys/p2pkey"
+
+type P2PKey struct {
+	id        string
+	peerID    string
+	publicKey string
+}
+
+func (k P2PKey) ID() string {
```

Comment:
```suggestion
func (k P2PKey) ID() graphql.ID {
```

Reply:
Good catch. Need to do the same for the OCR key queries and mutations. Will fix that in another PR.